lockbox-iosTests/WelcomePresenterSpec.swift

```diff
@@ -315,7 +316,7 @@ class WelcomePresenterSpec: QuickSpec {
                                 }
 
                                 it("starts authentication") {
-                                    expect(self.biometryManager.authMessage).to(equal(email))
+//                                    expect(self.biometryManager.authMessage).to(equal(email))
```

Comment:
Is this intentional, or a holdover from troubleshooting?

Reply:
holdover! good catch, thanks @linuxwolf